packages/forma-36-react-components/src/components/Card/AssetCard/AssetCard.css

```diff
@@ -44,11 +44,16 @@
   padding: 0 calc(1rem * (14 / var(--font-base-default)));
   height: calc(1rem * (45 / var(--font-base-default)));
   display: flex;
+  justify-content: flex-end;
```

Comment:
do we still need this?

Reply:
Without it:
<img width="319" alt="Screenshot 2020-11-24 at 11 24 49" src="https://user-images.githubusercontent.com/4171202/100081816-c05fed80-2e47-11eb-8d96-53f96c476d42.png">

With it:
<img width="300" alt="Screenshot 2020-11-24 at 11 24 57" src="https://user-images.githubusercontent.com/4171202/100081843-c6ee6500-2e47-11eb-8fb4-7cd628b62ad5.png">


Which way do we want to go @mshaaban0 ?

Reply:
Ah ok, I actually like the one on the left side, but let's keep the same convention 👍